client/packages/cli/src/commands/dgf.ts

```diff
@@ -0,0 +1,186 @@
+import ora from "ora"
+import { ApiPromise, WsProvider } from "@t3rn/sdk"
+import { getConfig } from "@/utils/config.ts"
+import { colorLogMsg } from "@/utils/log.ts"
+import {
+  ErrorListener,
+  ErrorMode,
+  ListenerEvents,
+  processSfx,
+} from "@/utils/dgf.ts"
+import { Extrinsic } from "@/schemas/extrinsic.ts"
+import { SfxSendType, submitSfx } from "@/utils/sfx.ts"
+import { Args } from "@/types.ts"
+
+const spinner = ora()
+const submitedRegistry = new Map<string, Extrinsic>()
+
+export const handleDgfCmd = async (
+  args: Args<"sfx" | "timeout" | "export">
+) => {
+  const config = getConfig()
+
+  if (!config) {
+    process.exit(1)
+  }
+
+  spinner.start(
+    colorLogMsg("INFO", "Starting the unhappy path generation process...")
+  )
+
+  try {
+    const provider = new WsProvider(getConfig().circuit.ws)
+    const api = await ApiPromise.create({ provider })
```

Comment:
This should come from SDK instance, as done in the other commands

Reply:
Fixed, refer to commit 884d656.